src/main/web_src/src/pages/tpsEndring/TpsEndring.js

```diff
@@ -5,9 +5,15 @@ import './TpsEndring.less'
 
 export default class TPSEndring extends PureComponent {
 	render() {
+		const dollySlack = <a href="https://nav-it.slack.com/archives/CA3P9NGA2">#dolly</a>
+
 		return (
 			<Fragment>
 				<h1>Send Endringsmelding</h1>
+				<p>
+					(Denne siden kommer til å bli fjernet. Kontakt oss på {dollySlack} hvis dette er et
+					problem for deg.)
```

Comment:
Kunne eventuelt forklart hva alternativet til å bruke denne siden er?

Reply:
Hva med å bruke Alertstripeinfo-komponenten for visning?

Reply:
Kan teste det ut

Reply:
![image](https://user-images.githubusercontent.com/1404881/91834351-363d2a00-ec48-11ea-8bc2-b2a01a2e5d7d.png)